tests/kafka/kafka.test.ts

```diff
@@ -170,4 +182,19 @@ class DBOSTestClass {
     }
     await DBOSTestClass.patternTopicPromise;
   }
+
+  @KafkaConsume(arrayTopics)
+  @Workflow()
+  static async testConsumeTopicsArray(_ctxt: WorkflowContext, topic: string, _partition: number, message: KafkaMessage) {
+    const isWfMessage = topic == wfTopic && message.value?.toString() === wfMessage;
+    const isTxnMessage = txnTopic && message.value?.toString() === txnMessage;
```

Comment:
Shouldn't this be topic === txnTopic?

Reply:
Sure! I just try to use the same code that is already there so as not to create friction